packages/joint-core/src/util/utilHelpers.mjs

```diff
@@ -301,7 +301,7 @@ const copyObject = (source, props, object) => {
     return object;
 };
 
-const isArrayLike = (value) => {
+export const isArrayLike = (value) => {
```

Comment:
Where is this used? I don't see any usages outside this file in either joint or joint plus